lib/include/openamp/remoteproc.h

```diff
@@ -411,6 +411,18 @@ struct remoteproc_ops {
 	int (*stop)(struct remoteproc *rproc);
 	int (*shutdown)(struct remoteproc *rproc);
 	int (*notify)(struct remoteproc *rproc, uint32_t id);
+	/**
+	 * notify_wait
+	 *
+	 * Wait for remote notified, when there is no TX buffer anymore.
+	 * Set to NULL means use usleep to wait TX buffer available.
+	 *
+	 * @rproc - pointer to remoteproc instance
+	 * @id    - the notifyid
+	 *
+	 * return 0 means there is notify available, otherwise negative value.
+	 */
+	int (*notify_wait)(struct remoteproc *rproc, uint32_t id);
```

Comment:
Here we don't wait that the remote is notified, but that the local is notified when the remote releases a buffer. 
So we wait for remote notification. your previous definition `wait_notified`or seems to me more explicit (  comment is valid for the whole commit)

Reply:
done